### ISSUE TITLE
Creation of a versioned "official docker image"

### ISSUE THREAD

**Issue by pat-s** [rocker-org/rocker-versioned#168]
Currently there is [one](https://hub.docker.com/_/r-base?tab=description) "official docker image" for R based on rocker/r-base.

I was wondering if there already have been thoughts on applying for a versioned one based on rocker/r-ver?

Instructions for applying for an official image can be found [here](https://docs.docker.com/docker-hub/official_images/#creating-an-official-image).

### Use case

I am currently [trying to add](https://github.com/circleci/circleci-images/pull/452) official support for the R language on Circle CI. There, images need to be based on a "official" docker image. This means currently no version support is possible. Hence the idea of a versioned "official docker image" for R.

**Comment by eddelbuettel** [comment]
There is.  Tags.

```sh
edd@rob:~/git/digest(master)$ cd
edd@rob:~$ docker images | grep r-base
r-base                       3.6.1               4e55790c88ae        3 months ago        642MB
r-base                       latest              4e55790c88ae        3 months ago        642MB
rocker/r-base                latest              b39219378c26        3 months ago        644MB
r-base                       3.6.0               6dfc0dec6d66        5 months ago        662MB
r-base                       3.5.3               62c848eeb175        7 months ago        649MB
r-base                       3.5.2               578644a0540c        9 months ago        650MB
r-base                       3.5.1               93ca38345db2        15 months ago       678MB
r-base                       3.5.0               fb43e8fe1667        16 months ago       698MB
r-base                       3.4.3               d1e1c25485af        19 months ago       670MB
r-base                       3.4.2               02d3b7e00020        23 months ago       651MB
r-base                       3.4.0               5a6c58403310        2 years ago         656MB
r-base                       3.3.3               88436550cddc        2 years ago         635MB
r-base                       3.3.1               7ba1baf9d8bb        3 years ago         657MB
edd@rob:~$
```

Good enough?

**Comment by eddelbuettel** [comment]
After all we are doing this for R,  and R and CRAN really 'live at @HEAD' (to borrow that phrase).  Current R-release matter, R-devel matters (and we have both in Rocker).  

Everything else can be used in Circle CI to everybody's heart's contents but it really does not have to be an official image, does it?   

@tianon @yosifkit have been approving my PR for the 'current at the time of release' r-base.  I don't think we need to go beyond this especially as we already have the entire r-versioned stack for R in this Rocker Project thanks to @cboettig . 

But it is really the call of @tianon @yosifkit.  There is already _a lot_ in the [official-images](https://github.com/docker-library/official-images/) repo but I no good feeling if beyond the one 'wide' dimentions of languages / frameworks / tools there is also a 'vertical' one for time...

**Comment by pat-s** [comment]
After reading Circle CIs policy again and exploring the tags of the r-base image, I think everything is fine.

Using r-base as the base for the Circle-CI image will create all tags that come with the r-base image. Hence all versions supported by r-base will also be supported by the Circle image.

My goal is to have an image for R [here](https://github.com/circleci-public/circleci-dockerfiles).
This would result in an instant startup when specifying circleci/r-base:<version> rather than having to pull the rocker images on every build (as it is now).
This saves both bandwith and time.

Thanks Dirk. I simply did not see the "tags" in the official docker image / was not aware of them previously (even though I used them, lol).
